docs/dev-guide/index.md

```diff
@@ -69,6 +69,11 @@ One of the biggest issues is the **scope of licenses**, namely what to do about
 - better user experience (otherwise all info must be transported via stdout)
 - to be sure nothing is manipulated within the validation script
 
+### Why is the data not in the image (inner contaienr) but in the outer container
```

Comment:
Not to be nitpicky, but there is a typo in the word container (inside the brackets).